foreman-apb/foreman/secrets/README.md

```diff
@@ -0,0 +1,50 @@
+make-certs.sh
+-------------------
+
```

Comment:
This `secrets/README.md` is good. It explains what the scripts do. I mentioned adding more detail to the main README.md but I think this one does what I was looking for.